exit_trust_root/src/bin.rs

```diff
@@ -25,58 +33,132 @@ pub const DOMAIN: &str = if cfg!(test) || DEVELOPMENT {
 /// The backend RPC port for the info server fucntions implemented in this repo
 const SERVER_PORT: u16 = 9000;
 
+lazy_static! {
+    static ref EXIT_CONTRACT_CACHE: Arc<RwLock<HashMap<Address, ExitContractSignatureCacheValue>>> =
+        Arc::new(RwLock::new(HashMap::new()));
```

Comment:
you can use actix_web db data for this (which won't run into scoping problems in the test environment) https://docs.rs/actix-web/latest/actix_web/web/struct.Data.html